mobile-app/cypress/e2e/functional/wallet/loans/loans.spec.ts

```diff
@@ -112,6 +123,16 @@ context("Wallet - Loans - Take Loans", () => {
     cy.getByTestID("loans_tabs_YOUR_VAULTS").click();
   });
 
+  it("should show correct token on browse loan token screen on active vault on search", () => {
```

Comment:
can also add test for no results

Reply:
Need more fine grained testing on search. I have updated the implementation to be consistent with other search component